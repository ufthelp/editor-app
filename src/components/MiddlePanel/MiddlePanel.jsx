import React from 'react';
import PropTypes from 'prop-types';
import Text from '../Text';
import './MiddlePanel.scss';


const MiddlePanel = ({ content }) => (
        <div className="middlePanelCta">
            <Text content={content} />
        </div>
    );


MiddlePanel.propTypes = {
    content: PropTypes.string.isRequired,
}

export default MiddlePanel;
