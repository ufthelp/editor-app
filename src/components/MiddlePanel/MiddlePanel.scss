@import "../../scss/variables.scss";

.middlePanelCta {
    background-color: $middlePanelbgColor;
    width: $middlePanelWidth;
    color: $mainTextColor
  }