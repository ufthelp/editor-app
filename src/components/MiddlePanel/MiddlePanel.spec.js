import React from 'react';
import {
    mount
} from "enzyme";
import MiddlePanel from './MiddlePanel';

describe('<MiddlePanel />', () => {
            let wrapper;
            beforeEach(() => wrapper = mount( < MiddlePanel content = {
                    "EXPLORER"
                }
                visibility
                /> ));

                it('should render correctly', () => expect(wrapper).toMatchSnapshot());

                it('should render <Text />', () => {
                    expect(wrapper.exists()).toBe(true)
                })

                it('Props is set to EXPLORER', () => {
                    expect(wrapper.prop('content')).toEqual('EXPLORER')
                })
            })