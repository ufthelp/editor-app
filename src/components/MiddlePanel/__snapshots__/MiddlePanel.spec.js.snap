// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MiddlePanel /> should render correctly 1`] = `ReactWrapper {}`;
